libs/s25main/ingameWindows/iwBuildings.cpp

```diff
@@ -67,10 +67,10 @@ iwBuildings::iwBuildings(GameWorldView& gwv, GameCommandFactory& gcFactory)
         }
     }
 
-    // Hilfe-Button
+    // "Help" button
```

Comment:
Just a note for the future: Instead of the work of translating such comments you can simply remove those that don't provide additional information: `AddImageButton(..., "Help")` doesn't need such a comment, especially when the ID is replaced by the enum-IDs (e.g. `ID_btHelp`)